app/models/commits_service.rb

```diff
@@ -0,0 +1,8 @@
+class CommitsService
+
+  def self.find_commits(username, repository)
+    response = Faraday.get("https://api.github.com/repos/#{username}/#{repository}/commits?client_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
```

Comment:
@Dpalazzari definitely. that makes sense. we can for sure talk about that!

rather than passing three arguments in, what about just passing in `current_user`? then, you'd have the username and the token as part of that object 👍